run_route_scripts/run_with_server.py

```diff
@@ -96,6 +97,9 @@ def make_request(post_body):
     shutil.rmtree(parsed_args.output_dir)
   os.mkdir(parsed_args.output_dir)
 
+  # setup http headers
+  parsed_args.headers = { k: v for k, v in [ h.split(': ') for hs in parsed_args.headers for h in hs] }
```

Comment:
so the cool thing about `action='append'` above is that it lets you specify the arg more than once. so you can call the program like:

    ./run_with_server.py --headers "x-my-custom-header: foo" --headers "x-my-other-custom-header: bar" ...

And what comes out in python is:

    [["x-my-custom-header: foo"], ["x-my-other-custom-header: bar"]]

In the end the `requests` module wants it in a dict so i unroll the lists of lists, split on the separator and do a dict comprehension to turn it into the required dict